scripts/generate_hillshade.py

```diff
@@ -194,20 +183,22 @@ def main() -> None:
 
     if arguments.collection_id:
         for path, derived_from_tiffs in tiles:
-            if path is None:
-                continue
-            # Create STAC and save in target
-            item = create_item(
-                asset_path=path,
-                start_datetime="",
-                end_datetime="",
-                collection_id=arguments.collection_id,
-                gdal_version=gdal_version,
-                current_datetime=arguments.current_datetime,
-                gdalinfo_result=None,
-                derived_from=[url_derived_from.rsplit(".", 1)[0] + ".json" for url_derived_from in derived_from_tiffs],
-            )
-            write(path.rsplit(".", 1)[0] + SUFFIX_JSON, dict_to_json_bytes(item.stac), content_type=ContentType.GEOJSON.value)
+            stac_item_path = path.rsplit(".", 1)[0] + SUFFIX_JSON
+            if not exists(stac_item_path):
+                # Create STAC and save in target
+                item = create_item(
+                    asset_path=path,
+                    start_datetime="",
+                    end_datetime="",
+                    collection_id=arguments.collection_id,
+                    gdal_version=gdal_version,
+                    current_datetime=arguments.current_datetime,
+                    gdalinfo_result=None,
+                    derived_from=[url_derived_from.rsplit(".", 1)[0] + ".json" for url_derived_from in derived_from_tiffs],
```

Comment:
nit - .json -> SUFFIX_JSON
```suggestion
                    derived_from=[url_derived_from.rsplit(".", 1)[0] + SUFFIX_JSON for url_derived_from in derived_from_tiffs],
```